.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
               gcloud --quiet container clusters get-credentials core-scaling --zone=europe-west3-a
               ./run.sh update
               POD=$(kubectl get pods --selector="role=nfs-server" -o=jsonpath='{.items[0].metadata.name}')
-              kubectl exec $POD -- chown -R 65534:65534 /exports/default
+              kubectl exec $POD -- chown -R 1910:1910 /exports/default
```

Comment:
An idea, could we do `kubectl exec $POD -- id -u` to get the container userid? That would work for any future changes too...

Reply:
That would mean that we have to do an exec into engine to get user and group, and then assign that in the chown. If engine where to change this often I would agree, but I don't see that as an issue today.

Reply:
That's what you're doing with `chown` above already :)

`id -u` would always work (e.g. if we rollback to an old engine with `nobody` on master, or someone is using this project with an older engine deployed) so thought it would be nice if someone were to copy/paste it elsewhere...

Reply:
Nevermind, we're exec:ing the nfs-server. Maybe the fever isn't gone 😅